compaction/task_manager_module.cc

```diff
@@ -334,6 +334,7 @@ static future<bool> maybe_flush_all_tables(sharded<replica::database>& db) {
     if (interval > 0s) {
         auto when = db_clock::now() - interval;
         if (co_await replica::database::get_all_tables_flushed_at(db) <= when) {
+            tasks::tmlogger.info("Flushing commitlog and all tables");
```

Comment:
This is inaccurate since the commit isn't flushed per-se.
We forcem a new commitlog segment and then flush all tables so the previously held commitlog segments get recycled.

The log message can say that "Forcing new commitlog segment and flushing all tables".

That said, why not log it in `flush_all_tables`?
It is in a better position to say what it is doing, and if we care about this log message, then we should see it also from other call sites.

Reply:
Ack. I have changed the log message and moved it inside flush_all_tables.